### PR TITLE
Gherkin scenarios for IS plugin

### PR DESCRIPTION
**Pull request by radinamatic**
Ready for review!

### CODE REVIEW

File: kolibri_instant_schools_plugin/integration-testing/vf-is-reset-password.feature [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Feature: Users need to be able to reset the password needed to access IS Kolibri
+
+  Background:
+    Given that I have an account for IS Kolibri with a mobile number <mobile-number>
+      And I am on the Kolibri sign in page
+      And I fogot the password <password> used to access my account
```
Comment (mrpau-richard) [inline kolibri_instant_schools_plugin/integration-testing/vf-is-reset-password.feature]: I think `fogot the password` is `forgot the password`

File: kolibri_instant_schools_plugin/integration-testing/vf-is-view-about-faq.feature [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    Scenario: View the About page
+      When I open the sidebar menu
+        And I click the *About* option
+      Then I am on the the *About* page
+
+    Scenario: View the FAQ page
+      Given that I am on the the *About* page
+        When I click the *More information* button
+        Then I am on the the *FAQ* page
+        When I click the *Bacl to About* button
```
Comment (mrpau-richard) [inline kolibri_instant_schools_plugin/integration-testing/vf-is-view-about-faq.feature]: I think the `Bacl` is a typo error @radinamatic

Comment (radinamatic) [reply kolibri_instant_schools_plugin/integration-testing/vf-is-view-about-faq.feature]: Thanks, corrected!

File: kolibri_instant_schools_plugin/integration-testing/vf-is-reset-password.feature (hunk 1 above)
Comment (mrpau-richard) [reply kolibri_instant_schools_plugin/integration-testing/vf-is-reset-password.feature]: Check this also @radinamatic ^^

**Review by mrpau-richard — APPROVED** [review]

**Pull request merged**